<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="apples-MacBook-Air.local" ignored="4" name="Default test(failed)" tests="1" failures="1" timestamp="2022-02-16T22:15:57 IST" time="0.066" errors="0">
  <testcase name="test5" time="0.011" classname="TestCases.DemoTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.DemoTest.test5(DemoTest.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
  <testcase name="test4" time="0.0" classname="TestCases.DemoTest">
    <ignored/>
  </testcase> <!-- test4 -->
  <testcase name="test3" time="0.0" classname="TestCases.DemoTest">
    <ignored/>
  </testcase> <!-- test3 -->
  <testcase name="test2" time="0.0" classname="TestCases.DemoTest">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase name="test1" time="0.0" classname="TestCases.DemoTest">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
